docs/topics/multiplatform-mobile/multiplatform-mobile-wrap-up.md

```diff
@@ -0,0 +1,43 @@
+[//]: # (title: Wrap up your project)
+
+<microformat>
+    <p>This is a part of the <strong>Getting started with Kotlin Multiplatform Mobile</strong> tutorial. Before proceeding, make sure you've completed previous steps:</p>
+    <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="multiplatform-mobile-setup.md">Set up environment</a><br/><img src="icon-2-done.svg" width="20" alt="Second step"/> <a href="multiplatform-mobile-create-first-app.md">Create your first cross-platform app</a><br/><img src="icon-3-done.svg" width="20" alt="Third step"/> <a href="multiplatform-mobile-dependencies.md">Add dependencies</a><br/><img src="icon-4-done.svg" width="20" alt="Fourth step"/> <a href="multiplatform-mobile-update-app.md">Upgrade your app</a><br/><img src="icon-5.svg" width="20" alt="Fifth step"/> <strong>Wrap up your project</strong></p>
+</microformat>
+
+You've created your first Multiplatform Mobile app that works both on iOS and Android. Now you know how
+to set up the environment for cross-platform mobile development, create a project in Android Studio, run your app on devices,
+and expand its functionality.
+
+Once you've gained some experience with Kotlin Multiplatform Mobile, you can take a look at some advanced topics and solve
+particular cross-platform mobile development tasks:
+
+## Dive deep into Kotlin Multiplatform Mobile
+
+* [Make your Android app cross-platform](multiplatform-mobile-integrate-in-existing-app.md)
```

Comment:
The reader just created a cross-platform app, we shouldn't provide him irrelevant information as the first step here